src/fava/beans/abc.py

```diff
@@ -112,7 +112,7 @@ def price(self) -> Amount | None:
 
     @property
     @abstractmethod
-    def meta(self) -> Meta | None:
+    def meta(self) -> Meta:
```

Comment:
this change should be reverted - we still need to support postings with a `None` meta.

Reply:
This should be fixed in https://github.com/beancount/fava/pull/1722/commits/33d1c8b094cd5115ff90fa3a29dd71ad70f80faa :)